src/utils/settings/splitFilters.js

```diff
@@ -0,0 +1,111 @@
+import { STANDALONE_MODE } from '../constants';
+import { uniq, isString } from '../lang';
+import logFactory from '../logger';
+const log = logFactory('splitio-settings');
+
+/**
+ * Validate, deduplicate and sort a given list of filter values.
+ * Exposed for testing purposes.
+ *
+ * @param {string} filterName string used for log messages
+ * @param {string[]} list list to validate, deduplicate and sort
+ * @param {number} maxLength
+ * @returns undefined or a sanitized list of values. The list is a non-empty array of unique and alphabetically sorted non-empty strings.
+ * @throws Error if the sanitized list exceeds the length indicated by `maxLength`
+ */
+function filterValidation(filterName, list, maxLength) {
+  // remove invalid values (no strings and empty strings)
+  let result = list.filter(value => {
+    if (!isString(value) || value === '') {
```

Comment:
You can just check value first, given empty string is falsey.

Reply:
it is done this way to filter also values that are not strings